lib/build.py

```diff
@@ -356,11 +356,18 @@ def __init__(self, *args):
          paths = [variables_sub(i, env.env_build)
                   for i in ch.tree_child_terminals(self.tree, "copy_shell",
                                                    "WORD")]
-         self.srcs = paths[:-1]
-         self.dst = paths[-1]
-      else:
-         assert (ch.tree_child(self.tree, "copy_list") is not None)
-         self.unsupported_yet_fatal("list form", 784)
+      elif (ch.tree_child(self.tree, "copy_list") is not None):
```

Comment:
Because this conditional is selecting between two options, a plain `elif` isn't really appropriate — what happens if neither is true? There are a couple ways to deal with this that I can think of: (1) the prior `else`/`assert` structure, or `if`/`elif`/`else` with the `else` asserting about unreachable code reached.